Packs/CommonScripts/Scripts/IsIPPrivate/IsIPPrivate.py

```diff
@@ -0,0 +1,111 @@
+from ipaddress import IPv4Address, IPv4Network
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+DELIMITER = ","
+
+
+def is_ip_internal(ip: str, ranges: List[str]) -> bool:
+    try:
+        return any((IPv4Address(ip) in IPv4Network(cidr.split(DELIMITER)[0] if DELIMITER in cidr else cidr) for cidr in ranges))
+    except ValueError:
+        demisto.info("One or more IP ranges or IPs are invalid. Please make sure the list is in the correct structure.")
+        return True
+
+
+def get_ip_tag(ip: str, ranges: List[str]) -> str:
+    for cidr in ranges:
+        tag = None
+        if DELIMITER in cidr:
+            cidr, tag = cidr.split(DELIMITER)[0], cidr.split(DELIMITER)[1].strip()
+        try:
+            if IPv4Address(ip) in IPv4Network(cidr):
+                return tag or ''
+        except ValueError:
+            if ip == cidr:
+                return tag or ''
+    return ''
+
+
+def main():
+    args = demisto.args()
+    ranges_list_name = args.get("PrivateIPsListName", "PrivateIPs")
+    ip_addresses_to_check = argToList(args.get("IPAddresses"))
+
+    # Get the list of private IP ranges from the XSOAR list:
+    private_ranges = demisto.executeCommand("getList", {"listName": ranges_list_name})[0]['Contents']
+    if "Item not found" in private_ranges:
+        return_error(f"The list name {ranges_list_name} does not exist.")
+
+    # Split ranges from XSOAR list to be a list:
+    if private_ranges:
+        try:
+            private_ranges = private_ranges.split("\n")
+        except Exception:
+            return_error(
+                "Could not parse the private ranges list. "
+                "Please make sure that the list contains ranges written in CIDR notation, separated by new lines.")
+    else:
+        private_ranges = ["10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"]  # No ranges in list, use default ranges
+        demisto.info(f"The list {ranges_list_name} was empty. Using the default private ranges as fall-back.")
```

Comment:
```suggestion
        demisto.debug(f"The list {ranges_list_name} was empty. Using the default private ranges as fall-back.")
```
Unless the log is really important, we always prefer debug logs (to reduce the number of unnecessary logs). Please change everywhere

Reply:
ok